src/ApiCalendar.js

```diff
@@ -188,13 +188,15 @@ class ApiCalendar {
      * Create Calendar event
      * @param {string} calendarId for the event.
      * @param {object} event with start and end dateTime
+     * @param {string} sendUpdates Acceptable values are: "all", "externalOnly", "none"
      * @returns {any}
      */
-    createEvent(event, calendarId = this.calendar) {
+    createEvent(event, sendUpdates = false, calendarId = this.calendar) {
```

Comment:
Pass the argument as last argument to avoid breaking change please

Reply:
You are saying sendUpdates can be equal to "all", "externalOnly" or "none" but you are making it equal to false by default.

Reply:
it is set false by default 

Whether to send notifications about the creation of the new event. Note that some emails might still be sent. The default is false.

Acceptable values are:
"all": Notifications are sent to all guests.
"externalOnly": Notifications are sent to non-Google Calendar guests only.
"none": No notifications are sent. This value should only be used for migration use cases (note that in most migration cases the import method should be used).

https://developers.google.com/calendar/v3/reference/events/insert